src/Butler/App/TodoManager.hs

```diff
@@ -256,14 +334,19 @@ newTask content taskPrio taskId = TodoTask taskId TaskNotSelected (TaskDesc cont
 
 setSelectedTask :: TodoManager -> TaskId -> TodoManager
 setSelectedTask (TodoManager{..}) taskId' =
-    TodoManager todoTaskIndex todoEditingTask $
-        map
+    TodoManager
+        todoTaskIndex
+        todoEditingTask
+        ( map
```

Comment:
nit: by changing the parameter order to `TaskId -> TodoManager -> TodoManager`, this function could be implemented with:

```suggestion
  #todoTasks %~ (map 
```